front/src/modules/simulationResult/components/ChartHelpers/enableInteractivity.tsx

```diff
@@ -282,8 +282,9 @@ export const enableInteractivity = <
     ])
     .wheelDelta(wheelDelta)
     // Allows evenements to be triggered on zoom and drag interactions for all graphs
-    .on('zoom', (event) => {
-      event.sourceEvent.preventDefault();
+    .on('zoom', (event: D3ZoomEvent<Element, ConsolidatedRouteAspect>) => {
+      const { sourceEvent }: { sourceEvent: Event } = event;
```

Comment:
Would be nicer to write `const sourceEvent: Event = event.sourceEvent`, unfortunately an ESLint rule forces destructuring.

Reply:
Seems like the zoom event is broken (if we zoom on space curve slopes, the position synchro with train details is lost) with the but it was the case before your PR :)